### PR TITLE
*: Update binary deployment

### PR DESCRIPTION
**Pull request by shenli**
@siddontang @ngaut PTAL

### CODE REVIEW

File: op-guide/binary-deployment.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ sha256sum -c tidb-latest-linux-amd64.sha256
 tar -xzf tidb-latest-linux-amd64.tar.gz
 cd tidb-latest-linux-amd64
 ```
-#### CentOS6
+#### CentOS6（不推荐）
```
Comment (qiuyesuifeng) [inline op-guide/binary-deployment.md]: Maybe  CentOS 6？

File: op-guide/binary-deployment.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 ## 下载官方 Binary
 
-### Linux 
+### Linux (CentOS7+, Ubuntu14.04+)
```
Comment (qiuyesuifeng) [inline op-guide/binary-deployment.md]: +1

**Comment by shenli** [comment]
@siddontang @qiuyesuifeng PTAL

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**